borg/remote.py

```diff
@@ -347,7 +360,7 @@ def load_key(self):
     def break_lock(self):
         return self.call('break_lock')
 
-    def close(self):
+    def _close(self):
```

Comment:
Yep, _close would now be a private method of Repository, which generally shouldn't be used from client code.